www/gatsby-config.js

```diff
@@ -342,7 +342,9 @@ module.exports = {
             headers.splice(1, 1)
           }
 
-          return headers
+          return headers.filter(
+            header => !header.toLowerCase().includes(`link:`)
```

Comment:
I admit that I'm not 100% sure about the implications of this change but I trust that it's correct.

Reply:
Also, they are guaranteed to be strings?

Reply:
Yes, as it's how netlify headers work every line in the array is a string